tests/lsdb/loaders/dataframe/test_read_dataframe.py

```diff
@@ -0,0 +1,40 @@
+import pandas as pd
+import pytest
+from hipscat.catalog import CatalogType
+
+import lsdb
+
+
+def test_read_catalog_from_dataframe(small_sky_order1_catalog, small_sky_order1_csv):
+    """Tests that we can initialize a catalog from a Pandas Dataframe
+    and that the loaded content is correct."""
+    # Read CSV file for the small sky order 1 catalog
+    catalog = lsdb.read_dataframe(
+        path=small_sky_order1_csv,
+        catalog_name="small_sky_order1",
+        catalog_type=CatalogType.SOURCE
+    )
+    assert isinstance(catalog, lsdb.Catalog)
+
+    # Catalogs have the same information
+    assert catalog.hc_structure.catalog_info == small_sky_order1_catalog.hc_structure.catalog_info
```

Comment:
The test data shouldn't ever change, but if you're comparing the catalog_info against the small_sky_order1 catalog info, you could make it so the catalog_name and catalog_type you use to initialize come from the small_sky_order1 catalog